src/20_Components/amp-reddit.html

```diff
@@ -0,0 +1,65 @@
+<!--
+  ## Introduction
+
+  The amp-reddit component allows embedding both reddit posts and comments into AMP files.
```

Comment:
can you please link to the official docs: 

```
[`amp-reddit`](...)
```

Reply:
Done.